pyproject.toml

```diff
@@ -8,17 +8,11 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = "^3.8"
 pytest = "^6.2.3"
```

Comment:
could probably move pytest to dev dependencies

Reply:
Good catch! Will do